src/settings/StaffSlips/StaffSlipForm.js

```diff
@@ -48,8 +48,7 @@ const StaffSlipForm = (props) => {
     : formatMessage({ id: 'ui-circulation.settings.staffSlips.new' });
 
   const footerPaneProps = {
-    pristine,
-    submitting,
+    disabled: pristine || submitting || !stripes.hasPerm('ui-circulation.settings.edit-staff-slips'),
```

Comment:
FOLIO's approach is to not show an element if a user doesn't have permission.

Reply:
Thank you, fixed (https://github.com/folio-org/ui-circulation/pull/1127/commits/ed0c3a77de8139e2c29216ec1b77a4a44e497d06) Could you please re-review?